contracts/marketplace/Marketplace.sol

```diff
@@ -187,6 +264,13 @@ contract Marketplace is Migratable, Ownable, Pausable {
       priceInWei,
       expiresAt
     );
+    emit AuctionCreated(
```

Comment:
I think it would be better in terms of gas usage to only emit the legacy event when calling the legacy method not always

Reply:
100% didn't realise